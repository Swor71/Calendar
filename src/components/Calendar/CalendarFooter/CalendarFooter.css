.calendar-footer-wrapper {
  margin-top: 20px;
  display: flex;
  flex-direction: column;
  font-size: .9em;
  font-weight: bold;
}
