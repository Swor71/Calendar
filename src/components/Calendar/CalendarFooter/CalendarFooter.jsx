import React, { PureComponent } from 'react';

class CalendarFooter extends PureComponent {
  render() {
    return (
      <div>

      </div>
    );
  }
}

export default CalendarFooter;
