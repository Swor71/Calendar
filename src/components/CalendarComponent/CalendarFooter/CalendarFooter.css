.calendar-footer__wrapper {
  margin-top: 20px;
  display: flex;
  flex-direction: column;
  font-size: .9em;
  font-weight: bold;
}
