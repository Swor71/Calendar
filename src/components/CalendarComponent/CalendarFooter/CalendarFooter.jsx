import React, { PureComponent } from 'react';

import './CalendarFooter.css';

class CalendarFooter extends PureComponent {
  render() {
    return (
      <div className='calendar-footer__wrapper'>
        <span>Minimum stay varies</span>
        <span>Updated 23 days ago</span>
      </div>
    );
  }
}

export default CalendarFooter;
