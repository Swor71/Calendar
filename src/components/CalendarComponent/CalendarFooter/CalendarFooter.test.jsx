import React from 'react';
import CalendarFooter from './CalendarFooter';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

describe('CalendarFooter', () => {
  it('renders the component', () => {
    const component = shallow(<CalendarFooter />);
    expect(component.length).toEqual(1);
  });

  it('renders correctly', () => {
    const component = renderer
      .create(<CalendarFooter />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
