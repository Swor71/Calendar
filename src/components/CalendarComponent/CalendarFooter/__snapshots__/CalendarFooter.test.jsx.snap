// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarFooter renders correctly 1`] = `
<div
  className="calendar-footer__wrapper"
>
  <span>
    Minimum stay varies
  </span>
  <span>
    Updated 23 days ago
  </span>
</div>
`;
